src/qt/clientmodel.cpp

```diff
@@ -84,7 +84,7 @@ QString ClientModel::getMasternodeCountString() const
 {
     // return tr("Total: %1 (PS compatible: %2 / Enabled: %3) (IPv4: %4, IPv6: %5, TOR: %6)").arg(QString::number((int)mnodeman.size()))
     return tr("Total: %1 (PS compatible: %2 / Enabled: %3)")
-            .arg(QString::number((int)mnodeman.size()))
+            .arg(QString::number((int)mnodeman.CountMasternodes()))
```

Comment:
```suggestion
            .arg(QString::number((int)mnodeman.CountMasternodes(0)))
```

otherwise you'd only count masternodes on 70210+

Reply:
Fixed